### ISSUE TITLE
Missing Cosmos v3.3 builds?

### ISSUE THREAD

**Issue by evansd** [jart/cosmopolitan#1111]
Hi there, I notice there's no v3.3 release directory here:
https://cosmo.zip/pub/cosmos/v/

And there's also no `cosmos-3.3.zip` in the release assets here:
https://github.com/jart/cosmopolitan/releases/tag/3.3

Is this just a case of "please wait, we only released this yesterday!" or has something else changed?

P.S. Thanks for one of the most exciting new software developments in recent years :)

**Comment by KyongSik-Yoon** [comment]
I'm also waiting cosmos!

**Comment by bjconlan** [comment]
This looks to be updated now. But I just installed it and when doing a `cosmocc --version` its still reading
```
cosmocc (GCC) 11.2.0
Copyright (c) 2023 Justine Alexandra Roberts Tunney
Cosmopolitan Libc and LLVM libcxx/compiler-rt are subject to non-GPL
notice licenses, e.g. ISC, MIT, etc. Your compiled programs must embed
our copyright notices. This toolchain is configured to do so default.
Cosmopolitan comes with absolutely NO WARRANTY of any kind.
For more information, see the Cosmopolitan LICENSE files.
Copyright (C) 2019 Free Software Foundation, Inc.
This launches GNU GCC/Binutils subprocesses, which is free software; see
cosmocc's LICENSE files for source code and copying conditions. There is
NO warranty; not even for MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
```

Which is unexpected given comments on the major changes (expecting gcc 12.3) as per release notes. (but again perhaps there is a better place to find the version information within the package that im unfamiliar with)

**Comment by jart** [comment]
Cosmos will be zipped up soon. With 3.3 the main goal was doing an atomic GCC upgrade. I'm still testing it and hammering out issues. Thanks for reporting the incorrect version being reported. That'll be fixed in 3.3.1.

**Comment by jart** [comment]
I've just cut a new release of cosmos which is available here:

- https://github.com/jart/cosmopolitan/releases/tag/3.3.1
- https://cosmo.zip/pub/cosmos/zip/cosmos-3.3.1.zip (all binaries)
- https://cosmo.zip/pub/cosmos/v/3.3.1/bin/ (individual binaries)

Enjoy!
